package.json

```diff
@@ -42,7 +42,7 @@
     "validate": "npm run style && npm run coverage"
   },
   "engines": {
-    "node": ">=6.11.2 <7",
+    "node": ">=6.11.2",
```

Comment:
Actually we support node `<11` see https://github.com/yvele/azure-function-express/pull/18#issuecomment-417846637

I think it's better to make it clear that the code has actually been tested for node `>=6.11.2 <11`

When the future node 12 version come out, we will do another PR with extended tests 😉

So at the moment I prefer `>=6.11.2 <11` over `>=6.11.2` to be more cautious and explicit.

Reply:
That's fine with me :)